src/components/app/list-new/AppList.tsx

```diff
@@ -604,6 +604,7 @@ export default function AppList() {
                     buttonText="new"
                     onClickCreateButton={handleCreateButton}
                     showCreateButton={serverMode === SERVER_MODE.FULL ? true : false}
+                    iconType="caret-down"
```

Comment:
Can we rename it to btnIconType or something so that it ll be relatable

Reply:
Done